// Copyright 2023 The ChromiumOS Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package modemloggerd;

message HelperEntry {
  string modem_name = 1;
  string filename = 2;
  repeated string extra_argument = 3;
  optional string output_dir_argument = 4;
}

message HelperManifest {
  repeated HelperEntry helper = 1;
}
